apps/chrome-devtools/src/extension/background.ts

```diff
@@ -114,12 +128,22 @@ const applyActivateState = async (appName: string, tabId: number) => {
 /** map of connection base on Tab ID */
 const connections = new Map<number, chrome.runtime.Port[]>();
 
+function postMessageToEveryone(content: string, type: typeof postMessageType) {
+  window.postMessage({
+    type: type,
+    to: 'app',
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment -- type of `content` is explicitly `any`
```

Comment:
Maybe you should think about removing the `@typescript-eslint/no-unsafe-assignment` rule, regarding the number of exceptions in this single small package :S